python/BioSimSpace/Sandpit/Exscientia/_SireWrappers/_residue.py

```diff
@@ -55,7 +56,7 @@ def __init__(self, residue):
         # Check that the residue is valid.
 
         # A Sire Residue object.
-        if isinstance(residue, _SireMol.Residue):
+        if isinstance(residue, _Sire.Mol._Mol.Residue):
```

Comment:
Why is `_Sire.Mol._Mol` needed in favour of `_SireMol`?

Reply:
It could be either, since the module imports both `import sire.legacy as _Sire` and `from sire.legacy import Mol as _SireMol`. It looks like `_Sire` isn't actually used anyway so this might be from when @chryswoods was working out whether the mixed-api worked. I will remove it and use `_SireMol` directly, as was done previously.